pkg/attest/platform_cert_fetcher.go

```diff
@@ -224,40 +237,48 @@ func (certFetcher CertFetcher) retrieveCertChain(chipID string, reportedTCB uint
 			vcekPEMBytes := pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: derBytes})
 
 			// now retrieve the cert chain
+			logrus.Info("Fetching cert chain from AMD endpoint...")
 			uri = fmt.Sprintf(AmdCertChainRequestURITemplate, certFetcher.Endpoint, certFetcher.TEEType)
 			certChainPEMBytes, err := fetchWithRetry(uri, defaultRetryBaseSec, defaultRetryMaxRetries)
 			if err != nil {
-				return nil, reportedTCB, errors.Wrapf(err, "pulling AMD certchain response from get request failed")
+				return nil, reportedTCB, errors.Wrapf(err, "pulling AMD cert chain response from get request failed")

```

Comment:
Context would be great, like from which URL did you pull?  This may already be captured in the function call.